### PR TITLE
MXZ Strong  Legacy Loccode change 20to21 2011-2019

### PR DESCRIPTION
**Pull request by salichon**
Legacy instrument site move  code 20 to 21 updates

### CODE REVIEW

File: install/streams.csv [hunk 1]
```diff
@@ -1281,10 +1281,11 @@ MWZ,21,200,false,false,false,2020-02-09T21:38:00Z,9999-01-01T00:00:00Z
...
 MXZ,20,1,false,false,false,2004-02-29T00:00:00Z,2011-08-17T22:50:00Z
-MXZ,20,50,false,false,false,2004-02-29T00:00:00Z,2019-11-01T00:05:00Z
-MXZ,20,200,false,false,true,2004-02-29T00:00:00Z,2019-11-01T00:05:00Z
-MXZ,21,50,false,false,false,2019-11-01T00:07:00Z,2020-06-16T00:00:00Z
-MXZ,21,200,false,false,true,2019-11-01T00:07:00Z,9999-01-01T00:00:00Z
+MXZ,20,50,false,false,false,2004-02-29T00:00:00Z,2011-08-17T22:50:00Z
+MXZ,20,200,false,false,true,2004-02-29T00:00:00Z,2011-08-17T22:50:00Z
+MXZ,21,50,false,false,false,2011-08-18T00:30:00Z,2020-06-16T00:00:00Z
+MXZ,21,200,false,false,true,2011-08-18T00:30:00Z,2020-02-08T23:59:00Z
+MXZ,21,200,false,false,false,2020-02-09T00:00:00Z,9999-01-01T00:00:00Z
```
Comment (elidana) [inline install/streams.csv:1288]: should the end/start date of this one be `2020-06-15` instead?
I can see miniseed "BN" file up to doy 2020/167 for this station

Comment (salichon) [reply install/streams.csv:1288]: Hello @elidana  this one states for  the (general) swap  from Triggered 200Hz to Continous  200hZ on 2020.040 ( 9 feb)
 (3rd logical exp.   `true` for Triggered changed for `false`)

**Review by mnaguit — APPROVED** [review]
LGTM 👍

**Comment by elidana** [comment]
Hi @mnaguit , sorry I did miss your review, will merge this now!

**Review by elidana — APPROVED** [review]

**Comment by salichon** [comment]
@mnaguit i resolved a conflict  between MXZ and LTZ in that reviewed delta PR 
Would you bless it ?  thanks!
j

**Pull request merged**